src/db/migrations/20190110043410-create-user.js

```diff
@@ -44,6 +44,10 @@ module.exports = {
       allowNull: false,
       type: Sequelize.BOOLEAN
     },
+    isActive: {
```

Comment:
Please create a new migration that will edit the users table and add this column to it. It's not healthy to be rolling back your migration just because you want to add a field.

Reply:
Ok, will do that.